VisualC-GDK/SDL/SDL.vcxproj.filters

```diff
@@ -126,6 +126,8 @@
     <ClCompile Include="..\..\src\misc\SDL_url.c" />
     <ClCompile Include="..\..\src\misc\windows\SDL_sysurl.c" />
     <ClCompile Include="..\..\src\power\SDL_power.c" />
+    <ClCompile Include="..\..\src\process\windows\SDL_process.c.c" />
+    <ClCompile Include="..\..\src\process\dummy\SDL_process.c.c" />
```

Comment:
Is it normal that both windows and dummy are included, and that the file names end with ".c.c"?

Reply:
I initially planned to conditionally enable SDL_Process for Gaming.Desktop.x64 only, and disable it for the Xbox platforms.
But I don't see how to do it without wrapping the c sources with big `#ifdef`s, and adding `SDL_PLATFORM_PROCESS` defines to all `SDL_build_config.h`'s.
We definitely need to do this, but we're still in early stages.